experiments/AtmosLES/cfsite_hadgem2-a_07_amip.jl

```diff
@@ -0,0 +1,706 @@
+    ts,
+    group_id,
+) where {FT}
+    # Boundary Conditions
+    u_star = FT(0.28)
+
+    problem = AtmosProblem(
+        boundaryconditions = (
+            AtmosBC(
+                momentum = Impenetrable(DragLaw(
+                    (state, aux, t, normPu_int) -> (u_star / normPu_int)^2,
+                )),
+                energy = PrescribedEnergyFlux((state, aux, t) -> hfls + hfss),
+                moisture = PrescribedMoistureFlux(
+                    (state, aux, t) ->
+                        hfls / latent_heat_vapor(param_set, ts),
+                ),
+            ),
+            AtmosBC(),
+        ),
+        init_state_prognostic = init_cfsites!,
+    )
+    model = AtmosModel{FT}(
+        AtmosLESConfigType,
+        param_set;
+        problem = problem,
+        turbulence = Vreman{FT}(0.23),
+        source = (
+            Gravity(),
+            LinearSponge(FT, zmax, zmax * 0.85, 1, 4),
+            LargeScaleProcess()...,
+            LargeScaleSubsidence()...,
+        ),
+        moisture = EquilMoist{FT}(; maxiter = 5, tolerance = FT(2)),
+        lsforcing = HadGEMVertical(),
+    )
+
+    # Timestepper options
+
+    # Explicit Solver
+    ex_solver = ClimateMachine.ExplicitSolverType()
+
+    # Multirate Explicit Solver
+    mrrk_solver = ClimateMachine.MultirateSolverType(
+        fast_model = AtmosAcousticGravityLinearModel,
+        slow_method = LSRK144NiegemannDiehlBusch,
+        fast_method = LSRK144NiegemannDiehlBusch,
+        timestep_ratio = 10,
+    )
+
+    # IMEX Solver Type
+    imex_solver = ClimateMachine.IMEXSolverType()
+
+    # Configuration
+    config = ClimateMachine.AtmosLESConfiguration(
+        forcingfile * "_$group_id",
+        N,
+        resolution,
+        xmax,
+        ymax,
+        zmax,
+        param_set,
+        init_cfsites!,
+        #ZS: multi-rate?
+        solver_type = imex_solver,
+        model = model,
+    )
+    return config
+end
+
+# Define the diagnostics configuration (Atmos-Default)
+function config_diagnostics(driver_config)
+    default_dgngrp = setup_atmos_default_diagnostics(
+        AtmosLESConfigType(),
+        "2500steps",
+        driver_config.name,
+    )
+    core_dgngrp = setup_atmos_core_diagnostics(
+        AtmosLESConfigType(),
+        "2500steps",
+        driver_config.name,
+    )
+    return ClimateMachine.DiagnosticsConfiguration([
+        default_dgngrp,
+        core_dgngrp,
+    ])
+end
+
+function main()
+
+    # Provision for custom command line arguments
+    # Convenience args for slurm-array launches
+    cfsite_args = ArgParseSettings(autofix_names = true)
+    add_arg_group!(cfsite_args, "HadGEM2-A_SiteInfo")
+    @add_arg_table! cfsite_args begin
+        "--group-id"
+        help = "Specify cfSite-ID for forcing data"
+        metavar = "site<number>"
+        arg_type = String
+        default = "site17"
+    end
+    cl_args =
+        ClimateMachine.init(parse_clargs = true, custom_clargs = cfsite_args)
```

Comment:
Since we're using `rand` in initialization, can we use `fix_rng_seed = true` here?

Reply:
Yes, should we do this for all the experiments using `rand` (in a separate PR)?

Reply:
If it is useful to run the experiment with a truly random seed at times, then `fix_rng_seed = true` should not be specified here. That way, you can run the experiment interactively (or in some other way) with true randomness without having to edit this file. For CI though, we don't want a random number possibly breaking things sometimes, so we specify this on the command line (in the buildkite file).

For experiments where it doesn't matter if it's truly random or not (such as ones run on the QA pipeline maybe?), you can add this parameter in the `init` parameter list.

Reply:
It doesn't really matter for this driver, although I will almost always use `fix_rng_seed = true` in my runscript for scientifically useful results (we compare several simulations and don't want a random number to be the reason of the difference). So maybe I will change it back?

Reply:
🤷‍♂️ That's up to you, I just wanted to point out the implications. :)

Reply:
If we set `fix_rng_seed = true` here, can we still use `fix_rng_seed = False` in the run script to run the experiment with true randomness? Or we will have to change the file?

Reply:
Yes, command line arguments override `init` parameters (if `parse_clargs = true` is specified, which it is, here).

Reply:
ok I will keep it as it is then, thanks Kiran!